scripts/rtx

```diff
@@ -103,51 +107,61 @@ function _itplugs() {
                 fi
                 ;;
             filemanager)
-                svn co https://github.com/nelu/rutorrent-thirdparty-plugins/trunk/filemanager > /dev/null 2>&1
-                chmod -R +x /srv/rutorrent/plugins/filemanager/scripts
-                cat > /srv/rutorrent/plugins/filemanager/conf.php << FMCONF
+                case $tag in
+                    3.*)
+                        svn co https://github.com/nelu/rutorrent-thirdparty-plugins/trunk/filemanager > /dev/null 2>&1
+                        chmod -R +x /srv/rutorrent/plugins/filemanager/scripts
+                        cat > /srv/rutorrent/plugins/filemanager/conf.php << FMCONF
 <?php
-
 \$fm['tempdir'] = '/tmp';
 \$fm['mkdperm'] = 755;
-
 // set with fullpath to binary or leave empty
 \$pathToExternals['rar'] = '$(which rar)';
```

Comment:
It's better to use `command -v` here instead of `which`. This is a posix command (shell built-in) works on every Linux distribution with no dependencies required. It also has less overhead than `which` and is not deprecated.

Reply:
We only support Debian/Ubuntu, so full posix compliance does not particularly concern me -- more understandable and readable code tends to be the language which I prefer and `which` is easier to understand for most than `command -v` (though there are a few `command -v`'s sprinkled in the project. As far as I'm aware, which was slated to be deprecated, but isn't going to be and the deprecation warning will be removed. 

https://lwn.net/Articles/874049/

Furthermore, I'm fairly certain the deprecation warnings only exist in Ubuntu short term releases right now, which we also don't support. If `which` does indeed get fully deprecated or moved out of debianutils then I will create a full PR which targets removing the use of `which` from the entire project.

Reply:
okay sounds good. This is really only a hack for 3.x anyways. Eventually, everyone will be on 4.x and there will be no need for it anymore. The file pathing was fixed in 4.x with `command -v`. Install scripts won't need to populate these.

Reply:
That's good to know -- I realized I didn't have to define the variables in the new FM install script. If they no longer need to be defined though then I'll go ahead and remove the `which` sections for the configuration of ruTorrent itself -- since install will only be supporting 4.0+ from this point